Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Fleet/Onboarding.hs

```diff
@@ -90,7 +90,7 @@ getOnboardingGetReferralDetails merchantShortId opCity fleetOwnerId referralCode
   merchant <- findMerchantByShortId merchantShortId
   merchantOpCity <- CQMOC.findByMerchantIdAndCity merchant.id opCity >>= fromMaybeM (MerchantOperatingCityNotFound $ "merchantShortId: " <> merchantShortId.getShortId <> " ,city: " <> show opCity)
   transporterConfig <- findByMerchantOpCityId merchantOpCity.id Nothing >>= fromMaybeM (TransporterConfigNotFound merchantOpCity.id.getId)
-  dr <- validateReferralCodeAndRole transporterConfig (Id fleetOwnerId) referralCode (Just FLEET_OWNER)
+  dr <- validateReferralCodeAndRole transporterConfig (Id fleetOwnerId) referralCode (Just OPERATOR)
```

Comment:
This code is confusing, could you rename `fleetOwnerId` to `requestorId` everywhere including spec?

And on dashboard side it would be more clear if you send `personId` form token, not `fleetOwnerId`

```
  let requestorId = apiTokenInfo.personId.getId
  Client.callFleetAPI checkedMerchantId opCity (.onboardingDSL.getOnboardingGetReferralDetails) requestorId referralCode
```